### PR TITLE
Add openshift

### PR DESCRIPTION
**Pull request by offlinehacker**

### CODE REVIEW

**Comment by jagajaga** [comment]
Please rename commits in your recent PRs as it said in `CONTRIBUTING.md`.

**Comment by jagajaga** [comment]
Please rename commits' messages in your recent PRs as it said in `CONTRIBUTING.md`.

**Comment by offlinehacker** [comment]
Will do in my next pull request, i didn't knew about these policy now, still thanks for letting me know.

**Comment by jagajaga** [comment]
You can just do `git rebase -i`, can't you? :)

**Comment by offlinehacker** [comment]
Done, merging

**Comment by jagajaga** [comment]
:smile:

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -12075,6 +12075,8 @@ let
 
   mutt-kz = callPackage ../applications/networking/mailreaders/mutt-kz { };
 
+  openshift = callPackage ../applications/networking/cluster/openshift { };
```
Comment (edolstra) [inline pkgs/top-level/all-packages.nix]: Name mismatch: the package is called `openshift` here but `openshift-origin` elsewehere.

**Pull request merged**